### PR TITLE
With this changes I was able to work with paths longer than 260 characters

### PR DESCRIPTION
**Pull request by thomasschroeder**
The corresponding registry key was already set without my interaction (Initial installed version: 1903, upgrades to 1909, 2004 and 20H2).
HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem\LongPathsEnabled == 1 (DWORD32)
The mentioned group policy from the link is called "Lange Win32-Pfade aktivieren" on a German Windows version (full path: Administrative Vorlagen\System\Dateisystem) but is still on not configured on my system - the registry entry is enough.

I have not tested it with VSS (never used it till now at all).

The necessary changes are described in detail:
https://docs.microsoft.com/de-de/archive/blogs/jeremykuhne/net-4-6-2-and-long-paths-on-windows-10

### CODE REVIEW

**Comment by asdfjkl** [comment]
Thanks for this patch.

I'll have to review this first, though. For VSS there is no .NET native access, hence one has to use AlphaVSS and AlphaFS (which wrap native Windows Calls). It's not clear what kind of errors the patch induces. It seems according to https://stackoverflow.com/questions/12747132/pathtoolongexception-c-sharp-4-5/12747217#12747217 one has to use  "\\?\" style in such a case.

In other words: Have some patience till I merge this :-)

**Comment by asdfjkl** [comment]
( added long path support in 2c0bf58783f62cd02724853c428e8fb84f05ae12 )
